### PR TITLE
fix: scroll on movile and backdrop added for menu

### PR DESCRIPTION
**Pull request by josanri**
Added backdrop to make background darker on drawer opened:
Fixed bug: Scrolling was allowed while drawer opened

### CODE REVIEW

File: src/components/Navigation.astro [hunk 1]
```diff
@@ -39,7 +40,7 @@ const featuredSponsors = DATA.sponsors.filter(
     max-xl:duration-0"
     tabindex="-1"
   >
-  <ul
+    <ul
```
Comment (davidrojom) [inline src/components/Navigation.astro:43]: max-xl:gap-8 is dupped

File: src/components/Navigation.astro [hunk 2]
```diff
@@ -190,22 +219,26 @@ const featuredSponsors = DATA.sponsors.filter(
 
 <style>
   @media not all and (max-width: 501px) {
-      .drawer-open-transition {
-        transition-duration: 500ms;
-        transform: translate(0, 0);
-      }
+    .drawer-open-transition {
```
Comment (davidrojom) [inline src/components/Navigation.astro:222]: what do you think of not negating mqs? just like this 

```css
  @media (min-width: 501px) {
    .drawer-open-transition {
      transition-duration: 500ms;
      transform: translate(0, 0);
    }
  }

  @media (max-width: 500px) {
    .drawer-open-transition {
      opacity: 1;
      transform: translate(0, 0);
      transition: opacity 0.3s ease-in-out, transform 0ms ease-in-out;
    }

    .drawer-close-transition {
      opacity: 0;
      transition: opacity 0.25s ease-in-out, transform 0ms ease-in-out 0.25s;
    }
  }
```

**Review by davidrojom — CHANGES_REQUESTED** [review]

File: src/components/Navigation.astro (hunk 2 above)
Comment (josanri) [reply src/components/Navigation.astro:222]: Yes, much better, I just got confused with some tailwind media-queries, changed at [latest commit](https://github.com/biznaga-fest/web/pull/10/commits/1c79b8e222120cddb4fead217f91b3b50f774d67)

File: src/components/Navigation.astro (hunk 1 above)
Comment (josanri) [reply src/components/Navigation.astro:43]: [Fixed](https://github.com/biznaga-fest/web/pull/10/commits/b41626f066c5ff9591d57004ec19dea1a202170a)

**Comment by davidrojom** [comment]
LGTM, merged

**Pull request merged**